test/test_distributions.py

```diff
@@ -467,8 +473,14 @@ def test_biject_to(constraint, shape):
     assert np.shape(actual) == batch_shape
     if len(shape) == transform.event_dim:
         if constraint is constraints.simplex:
-            expected = np.linalg.slogdet(jax.jacobian(transform)(x)[:-1, :])[1]
-            inv_expected = np.linalg.slogdet(jax.jacobian(transform.inv)(y)[:, :-1])[1]
+            expected = onp.linalg.slogdet(jax.jacobian(transform)(x)[:-1, :])[1]
```

Comment:
using `onp` because `slogdet` does not work in GPU

Reply:
How is this handled on the gpu? Does jax send the array to the CPU for the numpy operation and transfer the result back?

Reply:
In GPU, `onp.linalg.slogdet` returns a numpy array.

Reply:
Wouldn't that be an issue when we try to use that with a GPU based array?